From 1727cb88a183a9e787e359e42c0a499917ba495b Mon Sep 17 00:00:00 2001
From: Leah Rowe <leah@libreboot.org>
Date: Fri, 6 Oct 2023 23:17:06 +0100
Subject: [PATCH 1/1] add central Makefile in main directory

it just runs the Makefiles in build32 and build64

all, clean and distclean(runs clean) are supported

this makes memtest86plus integrate easier in lbmk

Signed-off-by: Leah Rowe <leah@libreboot.org>
---
 Makefile | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100644 Makefile

diff --git a/Makefile b/Makefile
new file mode 100644
index 0000000..50f2d58
--- /dev/null
+++ b/Makefile
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: 2023 Leah Rowe <leah@libreboot.org>
+
+all:
+	make -C build32
+	make -C build64
+
+clean:
+	make -C build32 clean
+	make -C build64 clean
+
+distclean:
+	make -C build32 clean
+	make -C build64 clean
-- 
2.39.2

